easybuild/easyconfigs/d/DBG2OLC/DBG2OLC-20170208-intel-2016b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'CmdCp'
+
+name = 'DBG2OLC'
+version = '20170208'
+commit = 'b452286'
+
+homepage = 'https://github.com/yechengxi/DBG2OLC'
+description = """DBG2OLC:Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation 
+ Sequencing Technologies"""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['https://github.com/yechengxi/DBG2OLC/archive/']
+sources = ['%s.tar.gz' % commit]
+
+cmds_map = [('.*', "$CXX $CXXFLAGS -o DBG2OLC *.cpp")]
```

Comment:
Because it's the proper thing to do. If you compile directly from source to binary, you need both `$CXXFLAGS` and `$LDFLAGS`. It's not because it works now that it will do so in the future.